book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Please pass the `person` who is renting the book and the `date` they rented the book.

See example:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```

Reply:
This was not addressed!

Reply:
- Kindly note that to achieve the Many-to-many association you have to pass the necessary parameters into the `add_rental ` method which you set in your rental class ` def initialize(date, person, book) `  so that when creating a new rental it should belong to a book and also a person and should have a date.
- so in your` person.rb` file the parameter person would be replaced with `self` which enables you to access the instance of the class person.